src/Article.jsx

```diff
@@ -58,6 +64,35 @@ function Article() {
     });
   };
 
+  const handleCommentSubmit = (event) => {
+    event.preventDefault();
+
+    if (!user) {
+      alert("Please login to post a comment.");
+      return;
+    }
+
+    if (!newComment.trim()) {
+      alert("Please enter a comment.");
+      return;
+    }
+
+    setIsSubmitting(true);
+
+    postComment(article_id, user.username, newComment)
+      .then((comment) => {
+        setNewComment("");
+        setPostSuccess(true);
```

Comment:
What is the reason for this state? You have comment as the parameter of the .then here which is coming from your post request but you are not doing anything with this.